src/screens/AddNewItemScreen/index.tsx

```diff
@@ -64,9 +91,33 @@ export const AddNewItemScreen: FC = () => {
     });
   }, [onPressBack, navigation]);
 
+  const onPressAdd = () => {
+    const newItem = createNewItemHelper(text, date);
+    createNotificationHelper({ channelId, newItem, date });
+    dispatch(addItemSagaAction({ newItem, userToken, callback }));
+  };
+
+  const onPressEdit = () => {
+    const { notificationId, id } = editItem!;
+    stopNotificationHelper(notificationId);
+    const newItem = createNewItemHelper(text, date);
+    createNotificationHelper({ newItem, date, channelId });
+    setLoading(true);
+    const modifiedItem = {
+      id,
+      text,
+      date: date.toString(),
```

Comment:
when I asked you create object, I meant about yout todoItem 
as here `addItemSagaAction({ newItem, userToken, callback }));`
you created `newItem` that includes all this fileds